apps/condo/migrations/20210913091712-0055_alter_user_email.js

```diff
@@ -0,0 +1,28 @@
+// auto generated by kmigrator
+// KMIGRATOR:0055_alter_user_email:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMi41IG9uIDIwMjEtMDktMTMgMDQ6MTcKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAwNTRfYXV0b18yMDIxMDkxMF8wOTMzJyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFsdGVyRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3VzZXInLAogICAgICAgICAgICBuYW1lPSdlbWFpbCcsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5UZXh0RmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgXQo=
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Alter field email on user
+--
+ALTER TABLE "User" DROP CONSTRAINT "User_email_key";
+DROP INDEX IF EXISTS "User_email_667201b5_like";
```

Comment:
Why we need to drop index, that is used for search optimisation?

Reply:
It was autogenerated - fixed